package.json

```diff
@@ -1,75 +1,75 @@
-    },
-    "license": "MIT",
-    "scripts": {
-        "clean": "node tools/build.js clean",
-        "copysample": "node tools/build.js copysample",
-        "dts": "node tools/build.js dts",
-        "tslint": "node tools/build.js tslint",
-        "normalize": "node tools/build.js normalize",
-        "pack": "node tools/build.js pack",
-        "pack:prod": "node tools/build.js packprod",
-        "b": "node tools/build.js clean normalize buildcommonjs",
-        "builddemo": "node tools/build.js builddemo",
-        "builddoc": "node tools/build.js builddoc",
-        "build": "node tools/build.js clean normalize tslint buildcommonjs dts pack:prod builddemo",
-        "build:ci": "node tools/build.js clean normalize tslint buildcommonjs buildamd dts pack packprod copysample builddemo builddoc",
-        "start": "webpack-dev-server",
-        "test": "karma start",
-        "test:debug": "karma start --no-single-run",
-        "publish": "node tools/build.js clean normalize tslint buildcommonjs buildamd dts pack packprod copysample builddemo publish",
-        "publish:dark": "node tools/build.js clean normalize tslint buildcommonjs buildamd dts pack packprod copysample builddemo publishdark",
-        "instrumentation": "cd publish/samplesite/instrumentation-tests && npm run instrumentation",
-        "instrumentation:chrome": "cd publish/samplesite/instrumentation-tests && npm run instrumentation:chrome",
-        "instrumentation:firefox": "cd publish/samplesite/instrumentation-tests && npm run instrumentation:firefox"
-    },
-    "devDependencies": {
-        "@types/jasmine": "2.6.0",
-        "@types/dom-inputevent": "1.0.5",
-        "glob": "7.1.4",
-        "husky": "^3.0.1",
-        "jasmine-core": "3.4.0",
-        "karma": "4.1.0",
-        "karma-chrome-launcher": "2.2.0",
-        "karma-jasmine": "2.0.1",
-        "karma-phantomjs-launcher": "1.0.4",
-        "karma-sourcemap-loader": "0.3.7",
-        "karma-webpack": "4.0.2",
-        "ncp": "2.0.0",
-        "prettier": "1.18.2",
-        "pretty-quick": "^1.11.1",
-        "progress": "2.0.3",
-        "rimraf": "2.6.3",
-        "source-map-explorer": "2.0.1",
-        "toposort": "2.0.2",
-        "ts-loader": "6.0.4",
-        "tslint": "5.18.0",
-        "tslint-eslint-rules": "5.4.0",
-        "tslint-microsoft-contrib": "6.2.0",
-        "typedoc": "0.8.0",
-        "typedoc-plugin-external-module-map": "1.0.0",
-        "typescript": "3.5.2",
-        "webpack": "4.35.2",
-        "webpack-cli": "3.3.5",
-        "webpack-dev-server": "3.7.2",
-        "@microsoft/loader-load-themed-styles": "1.7.166",
-        "@types/node": "12.0.10",
-        "@types/object-assign": "4.0.30",
-        "@types/react-dom": "16.8.4",
-        "css-loader": "3.0.0",
-        "node-sass": "4.12.0",
-        "sass-loader": "7.1.0",
-        "url-loader": "2.0.1"
-    },
-    "husky": {
-        "hooks": {
-            "pre-commit": "pretty-quick --staged"
-        }
-    }
-}
+{
```

Comment:
Don't change the line end to LF only.

Reply:
The file is now in CRLF right now (with this check-in), so this should not occur in the future.

Reply:
I'm sure that it was CrLf but this change makes it to be Lf.

Two evidences: 
1. File sizes: 3140 (after change) v.s. 3215 (before change), the diff is 75 bytes which equals the count of lines.
2. From a HEX viewer, I can see 0D0A before the change and 0A only after change.

Reply:
It was a bad config that was clobbering it post commit. I resolved it.